docs-website/package.json

```diff
@@ -39,7 +39,8 @@
     "markprompt": "^0.1.7",
     "react": "^18.2.0",
     "react-dom": "18.2.0",
-    "sass": "^1.43.2"
+    "sass": "^1.43.2",
+    "ts-node-dev": "^2.0.0"
```

Comment:
dont think this new dep is necessary

Reply:
Yup just updated generateDocsDir for the newer version of ts-node